NEWS

```diff
@@ -2,6 +2,57 @@
 XZ Utils Release Notes
 ======================
 
+5.4.1 (2022-01-10)
+
+    * liblzma:
+
+        - Created a workaround for a bug in CLMUL CRC64 feature for
+          MSVC (VS2015-2022).
+
+        - Fixed lzma_microlzma_encoder() posssibly returning a boolean
+          from the result of an inner function call. Now, it will
+          always return a proper lzma_ret value.
+
+        - Added a NULL check to lzma_index_hash_append() for consistency
+          with parameter validation done in lzma_index_append().
+
+        - Minor API documentation and internal header file updates.
+
+    * Build systems:
+
+        - Updated cmake_minimum_required from 3.13...3.16 to 3.13...3.25.
```

Comment:
I took this entry from the commit log, but I was wondering why we updated to 3.13...3.25. Since this says the changes won't affect this project, we need to give a reason why this update was done.